### DISCUSSION TITLE
News Anzeige

### DISCUSSION THREAD

**Discussion by wocky1968** [scheb/open-apexx#60] [General]
Hallo zusammen,

ich habe das Problem, dass News nur noch bis 31.12.2020 in der Übersicht angezeigt werden...
Es ist schon mega lange her, aber irgendwo musste man was eintragen, dass bis zu einem Datum die News angezeigt werden...
Ich finde es nicht und weiß auch nicht mehr wo das war... 
Kann mir jemand helfen? Das wäre mega lieb....

Habe gesehen, wenn ich das Veröffentlichungsdatum der News uf Dezember zurück setze dass es angezeigt wird....

Gruß
Frank

**Comment by bigfraggle** [comment]
Du meinst das hier?
https://github.com/scheb/open-apexx/issues/44

**Comment by wocky1968** [comment]
Das war es nicht, habe das Datum aber auch direkt angepasst....

Wenn ich eine News neu erstelle, dann muss ich das Datum im "Veröffentlichen ab" eingeben, das geht aber nur bis Ende 2020...
Das musste man auch anpassen, weiß aber nicht mehr wo, damit die News auch angezeigt wird...

**Comment by wocky1968** [comment]
Hier ein Screenshot
![Screenshot 2021-04-22 004058](https://user-images.githubusercontent.com/44481158/115630078-b6adb980-a303-11eb-9781-7bcf00dbab97.png)

**Comment by bigfraggle** [comment]
Ich muss mal nachher schauen... Bin mir aber eigentlich sicher, dass der Datumsbereich nur an der einen Stelle geändert werden muss.

**Comment by wocky1968** [comment]
Hat funktioniert.... Danke Dir !!!!!
Hab noch 3 weitere Homepages, da werde ich das auch direkt ändern.

**Comment by wocky1968** [comment]
Hallo Marc,

 

schaue hier : https://www.schuetzenverein-eller.de/news.php

In der News Neuanlage kann ich kein Datum Veröffentlichen ab nicht eintragen……

 

Gruß
Frank

 

Von: Marc ***@***.***> 
Gesendet: Donnerstag, 22. April 2021 00:18
An: scheb/open-apexx ***@***.***>
Cc: wocky1968 ***@***.***>; Author ***@***.***>
Betreff: Re: [scheb/open-apexx] News Anzeige (#46)

 

Du meinst das hier?
#44 <https://github.com/scheb/open-apexx/issues/44>

**Comment by bigfraggle** [comment]
Verstehe die Frage leider nicht... Sieht doch soweit alles okay aus.

**Comment by wocky1968** [comment]
Hallo Marc,ja danke. Nach dem Hinweis wo es zu ändern ist hat es funktioniert.Gruß FrankVon meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: Marc ***@***.***> Datum: 17.05.21  10:19  (GMT+01:00) An: scheb/open-apexx ***@***.***> Cc: wocky1968 ***@***.***>, Author ***@***.***> Betreff: Re: [scheb/open-apexx] News Anzeige (#46) 
Verstehe die Frage leider nicht... Sieht doch soweit alles okay aus.
